src/components/Button/Button.tsx

```diff
@@ -68,7 +72,7 @@ export interface Props {
   /**
    * Available _stylistic_ variations available for the Button component
    */
-  variant?: 'primary' | 'bare' | 'link' | 'table-header';
+  variant?: 'flat' | 'outline' | 'link' | 'plain';
```

Comment:
I also wonder ... if we can confirm with design on cutting down the styles, perhaps we could only have `variant` with no `color` prop, in which case defining tokens is less unwieldy. Do you have a sense of when that convo might be resolved?